library/src/main/java/com/novoda/downloadmanager/lib/DefaultNotificationInitialiser.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.NotificationManager;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+
+import com.novoda.downloadmanager.notifications.NotificationInitialiser;
+
+class DefaultNotificationInitialiser implements NotificationInitialiser {
```

Comment:
What do you think about renaming the interface to `NotificationCompatBuilderCreator`?